engine/apps/alerts/models/alert_receive_channel.py

```diff
@@ -230,10 +230,25 @@ def get_default_template_attribute(self, render_for, attr_name):
             defaults = getattr(self, f"INTEGRATION_TO_DEFAULT_WEB_{attr_name.upper()}_TEMPLATE", {})
         return defaults.get(self.integration)
 
+    class DuplicateDirectPagingError(Exception):
+        """Only one Direct Paging integration is allowed per team."""
+
+        DETAIL = "Direct paging integration already exists for this team"  # Returned in BadRequest responses
+
     @classmethod
     def create(cls, **kwargs):
+        organization = kwargs["organization"]
+        team = kwargs["team"] if kwargs["team"] else None
+        integration = kwargs["integration"]
+        if (
+            integration == AlertReceiveChannel.INTEGRATION_DIRECT_PAGING
+            and AlertReceiveChannel.objects.filter(
+                organization=organization, team=team, integration=integration
+            ).exists()
```

Comment:
This has been moved to .create() from .save() as we need to check for suplicates only during creation